threatgrid/ctia
# Add client id to stored entities

> **Epic** https://github.com/advthreat/iroh/issues/5710
> Related https://github.com/advthreat/iroh/issues/5714

- add client-id to Identity protocol and accordingly update jwt auth.
- add client-id to StoredEntity model and update realize entity fns.

<a name="qa">[§](#qa)</a> QA
============================

1. Create 2 IROH clients
2. Create an incident with jwt created with each client
==> check that created incidents contain the field `client-id` with the value of corresponding client.
3. for earch client: `GET /ctia/incident/search?query=client_id:{client_id}`
==> check that each response only return the incident created with corresponding client id

<a name="release-notes">[§](#release-notes)</a> Release Notes
=============================================================



```
intern: In CTIA, each newly created entity contains now the client-id. .
```
